tutorials/05_hybrid_model.ipynb

```diff
@@ -21,7 +21,7 @@
   },
```

Comment:
<p>Can we remove the double space before "... item metadata..."?</p><p> </p><p>Additionally, at the bottom of this text block, can we add text that says something like:</p><p> </p><pre class="ql-syntax" spellcheck="false">Note that while we pass both user _and_ item metadata to the hybrid model below, we would also be able to pass in _just_ user or _just_ item metadata too and still a valid model definition. This applies to all hybrid model architectures in Collie.
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ahuds001/collie/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>